edgedb/protocol/protocol.pxd

```diff
@@ -40,10 +40,11 @@ include "./codecs/codecs.pxd"
 ctypedef object (*decode_row_method)(BaseCodec, FRBuffer *buf)
 
 
-cpdef enum IoFormat:
+cpdef enum OutputFormat:
     BINARY = b'b'
     JSON = b'j'
     JSON_ELEMENTS = b'J'
+    NULL_ = b'n'
```

Comment:
Not a fan of trailing underscore here.  Let's use `NONE` to avoid reserved keyword use.